simple/rdm_shared_ctx.c

```diff
@@ -491,7 +491,7 @@ static int run(void)
 
 	run_test();
 	/* TODO: Add a local finalize applicable to shared ctx */
-	//ft_finalize(ep[0], scq, rcq, remote_fi_addr[0]);
+	//ft_finalize(ep[0], scq, rcq, remote_fi_addr[0], 0);
```

Comment:
Incorrect update here? (I know it's commented out right now, but should probably be fixed...)